crypto/ui/ui_openssl.c

```diff
@@ -563,11 +563,10 @@ static int noecho_console(UI *ui)
     return 1;
 }
 
-static int echo_console(UI *ui)
+static int restore_console(UI *ui)
 {
 #if defined(TTY_set) && !defined(OPENSSL_SYS_VMS)
     memcpy(&(tty_new), &(tty_orig), sizeof(tty_orig));
-    tty_new.TTY_FLAGS |= ECHO;
 #endif
 
 #if defined(TTY_set) && !defined(OPENSSL_SYS_VMS)
```

Comment:
While you're at it, would it be appropriate to remove the fold these two #if-s into one?